### PR TITLE
feature(camera): minor improvements

### PR DESCRIPTION
**Pull request by IllustrisJack**
- allow right click camera orbiting on stations

### CODE REVIEW

File: script/Systems/Camera/CameraChase.lua
Comment (Flatfingers) [inline script/Systems/Camera/CameraChase.lua:1]: Can we split the difference at 2.5? :D 

I actually don't mind raising the player's ship "up" just a little as I like to adjust the FOV to flatten out the screen a bit. That pulls the player's ship toward the camera, so having it "up" a little actually helps keep it onscreen.

File: script/Systems/Controls/Controls/DockControl.lua
Comment (Flatfingers) [inline script/Systems/Controls/Controls/DockControl.lua:1]: I wasn't able to tell what effect these changes had. I thought they might make the camera orbit the space station while the player's ship is docked, but nothing like that appeared to be happening.

File: script/Systems/Camera/CameraChase.lua
Comment (IllustrisJack) [reply script/Systems/Camera/CameraChase.lua:1]: sounds good!

File: script/Systems/Controls/Controls/DockControl.lua
Comment (IllustrisJack) [reply script/Systems/Controls/Controls/DockControl.lua:1]: you can now use right click to orbit the camera around the station ;)

File: script/Systems/Camera/CameraChase.lua
Comment (IllustrisJack) [reply script/Systems/Camera/CameraChase.lua:1]: we should also make this realtime configurable at some point!

**Review by Flatfingers — APPROVED** [review]

**Pull request merged**